install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1584,6 +1584,7 @@ hubble:
     # -- Enable prometheus metrics for hubble-relay on the configured port at
     # /metrics
     prometheus:
+      metricsService: false
```

Comment:
This is under `hubble.prometheus`, but the code above checks the top-level `prometheus` value. I think you mixed up the two.

Reply:
Made changes as requested.

Reply:
Thanks it seems you removed the Hubble value, but `.Values.prometheus.metricsService` does still not exist in `values.yaml.tmpl`, even though it is referenced by the service

Reply:
@gandro Sorry. I am not clear with what you are saying. I added metricsService in values.yaml.tmpl.

Reply:
I see the value for `.Values.operator.prometheus.metricsService` (note the `operator` int he middle), but not the value `.Values.prometheus.metricsService`. Since you are referencing both in the service templates, you need to define both.

Reply:
Thank you. I found it. Sorry for my mistake.

Reply:
@gandro Made changes.